OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -108,7 +111,7 @@ public override void Activate(Actor self, Order order, SupportPowerManager manag
 				wsb.PlayCustomAnimation(self, info.ActivationSequence, () => wsb.CancelCustomAnimation(self));
 			}
 
-			var targetPosition = self.World.Map.CenterOfCell(order.TargetLocation);
+			var targetPosition = self.World.Map.CenterOfCell(order.TargetLocation) + new WVec(WDist.Zero, WDist.Zero, info.AirburstAltitude);
```

Comment:
AirburstAltitude is a `WDist` and this was copied from the missile projectile from consistency.